[BUG] Error When Using get_funders.py

When I run:

```bash
python3 funderfinder/get_funders.py repo_name=georgetown-cset/funder-finder
```

I get:

```
Traceback (most recent call last):
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/get_funders.py", line 39, in <module>
    print(get_project_funders(args.repo_name))
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/get_funders.py", line 20, in get_project_funders
    funding = finder.run(repo_name)
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/sources/github_sponsors.py", line 174, in run
    top_contribs = self.get_gh_top_contributors_json(gh_project_slug)
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/sources/github_sponsors.py", line 93, in get_gh_top_contributors_json
    for contributor in contributors:
UnboundLocalError: local variable 'contributors' referenced before assignment
```

Am I doing something wrong? Is there really a bug? `pytest` passes when I run it. hmmm...

And now new error:

```
Traceback (most recent call last):
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/get_funders.py", line 39, in <module>
    print(get_project_funders(args.repo_name))
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/get_funders.py", line 19, in get_project_funders
    finder = finder_class()
  File "/Users/johnspeedmeyers/Desktop/repos/public/funder-finder/funderfinder/sources/opencollective.py", line 20, in __init__
    assert os.environ.get(
AssertionError: Please `export OPENCOLLECTIVE_API_KEY=<your opencollective api key>
```

So the documentation should also add information on adding an Open Collective API key. In fact, I don't know how to do that. I need to look into that.

--- a/README.md
+++ b/README.md
@@ -30,7 +30,7 @@ Install the dependencies:
 pip install -r requirements.txt
 ```
 
-Add a GitHub username and a [GitHub API token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as environment variables. For the API token, make sure to enable these scopes: `admin:org`, `read:user`, `repo`, `user:email`, and`workflow`.
+Add a GitHub username and a [GitHub API token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as environment variables. For the API token, make sure to enable these scopes: `admin:org`, `read:user`, `repo`, `user:email`, and `workflow`.
 
 ```bash
 export GITHUB_USERNAME=YOUR_GITHUB_USERNAME

--- a/README.md
+++ b/README.md
@@ -30,7 +30,7 @@ Install the dependencies:
 pip install -r requirements.txt
 ```
 
-Add a GitHub username and a [GitHub API token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as environment variables:
+Add a GitHub username and a [GitHub API token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as environment variables. For the API token, make sure to enable these scopes: `admin:org`, `read:user`, `repo`, `user:email`, and`workflow`.
 
 ```bash
 export GITHUB_USERNAME=YOUR_GITHUB_USERNAME

